.gitignore

```diff
@@ -33,3 +33,4 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/config/local_env.yml
```

Comment:
FYI: you can also use `dot_env` gem or `figaro` gem. Please learn about the same

Reply:
Sure!